site/troubleshooting.md

```diff
@@ -290,6 +291,27 @@ Proxy](/site/tasks/weave-docker-api/weave-docker-api.md) will re-attach any appl
 containers that it originally attached to the Weave network when they
 restart.
 
+## <a name="v2plugin"></a>Troubleshooting the V2 plugin
+
+If you have installed Weave Net via `docker plugin install`, then you
+will need to download the `weave` script to run `weave status`, `weave ps`
+or `weave report` as above.  Do this as described in the
+[install instructions](/site/install/installing-weave.md).
+
+Docker "v2" plugins run as containers, but at a lower level within the
+Docker environment, so you cannot view them with `docker ps`, `docker
+inspect`, etc.
+
```

Comment:
Docker "v2" plugins do run as containers, but at a much lower level within the
Docker environment. Because of this, you cannot view them with Docker commands like `docker ps`, `docker
inspect`, etc.